master/buildbot/test/unit/test_config.py

```diff
@@ -12,7 +12,10 @@
 # Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
 #
 # Copyright Buildbot Team Members
-import __builtin__
+try:
```

Comment:
I can't use the future package's implementation of the builtins module for py2,
because this test is doing funky stuff of patching the open() method to override it in a test.  I tried to use it and the test broke.